### PR TITLE
WaveMaker 10.6 blog

### PR DESCRIPTION
**Pull request by swethakundaram**
Draft-1 - do not merge yet.

### CODE REVIEW

File: website/blog/2020-12-08-wavemaker-release-10-6-version.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+With the Angular 9 upgrade, WaveMaker applications support AOT compilation by default, however, it continues to use View Engine for rendering instead of Ivy. We expect to support Ivy in the next major release WaveMaker-11.
+
+:::note
+With Angular 9 update, you can expect to see an increase in the application build time.
+:::
+
+### Up-to-date Libraries
+
+Libraries are hosted in the public npm registry (Angular library), and Maven (Jars). Therefore, all the required libraries are available to use without needing to import ZIP files manually.
```
Comment (krishnaraoch) [inline website/blog/2020-12-08-wavemaker-release-10-6-version.md:27]: this line is not relevent to update to date libs

File: website/blog/2020-12-08-wavemaker-release-10-6-version.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Restore page by using [Page Cache](/learn/app-development/ui-design/page-concepts/page-cache) and restore widget state or share URL of a specific state of the widget by using [Retain Widget State](/learn/blog/2020/11/09/Retain-UI-State-on-Wavemaker-Apps).
+
+## What's changed
+
+### Phonegap Support
+
+As Phonegap has stopped supporting the platform, therefore we developed our very own [AppChef app](/learn/hybrid-mobile/mobile-build-appchef) to generate Android and iOS applications for testing as well as to publish apps to the app stores.
+
+### External CI-CD for Web and Mobile
```
Comment (krishnaraoch) [inline website/blog/2020-12-08-wavemaker-release-10-6-version.md:53]: You can write this here
Libraries are hosted in the public npm registry (Angular library), and Maven (Jars). Therefore, all the required libraries are available to use without needing to import ZIP files manually.

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

**Comment by swethakundaram** [comment]
Added more items to the release notes and pointed them from the blog as pointers to avoid duplicates. Let me know if this is good?

**Review by sboyina — APPROVED** [review]
Mobile related changes are good

**Review by krishnaraoch — APPROVED** [review]

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

**Pull request merged**